File: app/src/main/java/fr/erban/dixitcompanion/game/turn/bean/VoteBean.java
package fr.erban.dixitcompanion.game.turn.bean;

import fr.erban.dixitcompanion.game.player.Player;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class VoteBean {

    private Player voter;

    private Player elected;
}

File: dixit-companion/src/main/java/fr/erban/dixitcompanion/game/turn/bean/VoteBean.kt
package fr.erban.dixitcompanion.game.turn.bean

import android.os.Parcelable
import fr.erban.dixitcompanion.game.player.PlayerBean
import kotlinx.parcelize.Parcelize

@Parcelize
data class VoteBean(
    val voter: PlayerBean,
    val elected: PlayerBean
) : Parcelable
